index.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Onomeh's Portfolio</title>
+    <link
+      rel="stylesheet"
+      href="https://fonts.googleapis.com/css?family=Poppins:regular,bold,500,600&amp;display=swap"
+        />
+      <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <div class="container">
+      
+   <a href="/" class="logo">My Logo</a>
+              <nav class="navbar">
+                <button class="nav-button" type="button">
+                  <img src="assets/Icons/hamburger-button.svg" alt="more" />
+                </button>
+              </nav>                      
+       
+      </div>
+    
```

Comment:
- [ ] [OPTIONAL] As it was not marked by the previous review, this is marked as an optional suggestion: 
- Please, could you put your toolbar section into the `header` tag - it is for considering HTML semantic structure 👍